packages/core/src/render3/deps_tracker/deps_tracker.ts

```diff
@@ -12,12 +12,20 @@ import {Type} from '../../interface/type';
 import {NgModuleType} from '../../metadata/ng_module_def';
 import {getComponentDef, getNgModuleDef, isStandalone} from '../definition';
 import {ComponentType, NgModuleScopeInfoFromDecorator} from '../interfaces/definition';
-import {verifyStandaloneImport} from '../jit/directive';
-import {isComponent, isDirective, isModuleWithProviders, isNgModule, isPipe} from '../jit/util';
+import {isComponent, isDirective, isNgModule, isPipe, verifyStandaloneImport} from '../jit/util';
 import {maybeUnwrapFn} from '../util/misc_utils';
 
 import {ComponentDependencies, DepsTrackerApi, NgModuleScope, StandaloneComponentScope} from './api';
 
+/**
+ * Indicates whether to use the runtime dependency tracker for scope calculation in JIT compilation.
+ * The value "false" means the old code path based on patching scope info into the types will be
+ * used.
+ *
+ * @deprecated For migration purposes only, to be removed soon.
+ */
+export const USE_RDT_FOR_JIT = false;
```

Comment:
nit: I think having a full name here would make it easier to read the code below where the flag is used:
```suggestion
export const USE_RUNTIME_DEPS_TRACKING_FOR_JIT = false;
```

Reply:
Done!